[MozillaSecurityBridge] Make the URI unique by adding timestamp

Nono-m0le opened the pull request:
Mandatory for some feed reader (Kriss Feed for instance)

Nono-m0le committed:
Make the URI unique by adding timestamp

Mandatory for some feed reader (Kriss Feed for instance)

logmanoriginal commented:
Thanks for this PR!

This PR is a workaround for the issues discussed in #977 but only works for one bridge. The proposed fix is to introduce a new (optional) parameter to `$item` that can be used for identifying items in the feed uniquely. This should be implemented in next few days (if I don't get lazy again).

I'll get back to you once it is implemented. Leave this PR open in the mean time.

logmanoriginal commented on bridges/MozillaSecurityBridge.php:
```diff
@@ -21,7 +21,7 @@ public function collectData(){
 			$item['timestamp'] = strtotime($element->innertext);
 			$item['content'] = $element->next_sibling()->innertext;
-			$item['uri'] = self::URI;
+			$item['uri'] = self::URI . '?' . $item['timestamp'];
```
#1017 recently got merged, so this should work now:

```suggestion
			$item['uri'] = self::URI;
			$item['uid'] = self::URI . '?' . $item['timestamp'];
```

Note: `$item['uid']` should receive a string that uniquely identifies the item (and never changes between calls).

Nono-m0le committed:
added unique UID + URI 

if UID is mandatory for RSS-Bridge, the unicity of the URI is also mandatory for some reader (like kriss feed).

Nono-m0le commented:
Hi @LogMANOriginal just for me to know, do I need to do anything special for this PR to be approved ?

logmanoriginal commented:
Thanks @teromene for stepping in. Sorry @Nono-m0le I've been ill the past weeks, so I couldn't merge.

Nono-m0le commented:
No problem at all, important is the community can now hopefully enjoy it :)
